Cloud changelog May 4 (#6351)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2023-05-04](#release-2023-05-04)
- [Release 2023-04-21](#release-2023-04-21)
- [Release 2023-03-29](#release-2023-03-29)
- [Release 2023-03-20](#release-2023-03-20)
- [Release 2023-01-26](#release-2023-01-26)
- [Release 2023-01-16](#release-2023-01-16)

## Release 2023-05-04

### Highlights
 - Added an improved Cloud trial experience in preparation for replacing the Cloud Free experience. See more details in these Forum posts:
    - https://forum.mattermost.com/t/cloud-monthly-billing-deprecation/15804
    - https://forum.mattermost.com/t/cloud-free-deprecation/15802

### Improvements

#### User Interface (UI)
 - Replaced the ``compass-components`` icon component with ``compass-icons``.
 - Added “hours ahead” timezone details to the user profile popover.
 - Pre-packaged Calls v0.15.1.

#### Administration
 - Removed the deprecated ``model.CommandArgs.Session``.

>  - The database section in the **System Console** now has an additional read-only section which shows the active search backend in use. This can be helpful to confirm which search engine is currently active when there are multiple of them configured.

Reviewer: ```suggestion
 - The database section in the **System Console** now has an additional read-only section which shows the active search backend in use. This can be helpful to confirm which search engine is currently active when there are multiple configured.
```

 
#### Performance
 - Improved the performance of webapp related to timezone calculations.
 - Improved performance of code used for post list screen reader support.
 
### API Changes
 - An underscore is now used in the timeline API (``event-id`` -> ``event_id``) for consistency with other API arguments.

### Bug Fixes
 - Fixed an issue where a user would still see threads in the threads view of channels they have left. Migration execution time in MySQL: Query OK, 2766769 rows affected (4 min 47.57 sec). Migration execution time in PostgreSQL: Execution time: 58.11 sec, DELETE 2766690.
 - Fixed an issue where clicking on a channel link (for a channel the user was not a part of) caused the webapp to refresh, dropping the user from a call.
 - Fixed an issue with PDF preview rendering for certain Japanese characters.

>  - Fixed an issue where the screen reader did not announce the link copied on copying the link in invite modal.

Reviewer: ```suggestion
 - Fixed an issue where the screen reader did not announce the action of copying the link in the invite modal.
```
